test/stache-test.js

```diff
@@ -18,6 +18,7 @@ var parser = require('can-view-parser');
 var nodeLists = require('can-view-nodelist');
 var canBatch = require('can-event/batch/batch');
 var makeDocument = require('can-vdom/make-document/make-document');
+var devHelpers = require('can-test-helpers/lib/dev');
```

Comment:
Why have this in a module? I think we should just put everything on the package.